extern/CMakeProject-png.cmake

```diff
@@ -3,31 +3,35 @@ if(WITH_SYSTEM_PNG)
   set(PNG_LIBRARIES ${PNG_LIBRARIES} PARENT_SCOPE)
 else()
   set(PNG_SRC
-      "libpng/include/png.c"
-      "libpng/include/pngerror.c"
-      "libpng/include/pngget.c"
-      "libpng/include/pngmem.c"
-      "libpng/include/pngpread.c"
-      "libpng/include/pngread.c"
-      "libpng/include/pngrio.c"
-      "libpng/include/pngrtran.c"
-      "libpng/include/pngrutil.c"
-      "libpng/include/pngset.c"
-      "libpng/include/pngtest.c"
-      "libpng/include/pngtrans.c"
-      "libpng/include/pngwio.c"
-      "libpng/include/pngwrite.c"
-      "libpng/include/pngwtran.c"
-      "libpng/include/pngwutil.c")
+      "libpng/png.c"
+      "libpng/pngerror.c"
+      "libpng/pngget.c"
+      "libpng/pngmem.c"
+      "libpng/pngpread.c"
+      "libpng/pngread.c"
+      "libpng/pngrio.c"
+      "libpng/pngrtran.c"
+      "libpng/pngrutil.c"
+      "libpng/pngset.c"
+      "libpng/pngtest.c"
+      "libpng/pngtrans.c"
+      "libpng/pngwio.c"
+      "libpng/pngwrite.c"
+      "libpng/pngwtran.c"
+      "libpng/pngwutil.c")
+
+  configure_file("libpng/scripts/pnglibconf.h.prebuilt"
+                 "libpng/pnglibconf.h"
+                 COPYONLY)
```

Comment:
In the old tree, this was done manually and committed in git. We now do this while building.